src/main/java/com/arcanewarrior/disguises/MinestomDisguises.java

```diff
@@ -1,35 +1,43 @@
 package com.arcanewarrior.disguises;
 
 import com.arcanewarrior.disguises.commands.CommandInitializer;
-import com.arcanewarrior.disguises.events.DisguiseEvents;
 import com.arcanewarrior.disguises.events.EventInitializer;
 import net.minestom.server.MinecraftServer;
-import net.minestom.server.extensions.Extension;
 import net.minestom.server.tag.Tag;
+import xyz.citywide.citystom.Extension;
```

Comment:
Depending on how much of the library is used, it may be removed in the future. If all we're using is a single convenience method from it, I'd rather just copy and paste that convenience method.